lib/net/ssh/command_stream.rb

```diff
@@ -60,8 +60,13 @@ def initialize(ssh, cmd = nil, pty: false, cleanup: false)
         self.lsock.localinfo = "#{info[1]}:#{info[2]}"
 
         rssh.open_channel do |rch|
+          unless rch
```

Comment:
I'm not sure this will work yet. Please hold.

Reply:
(I doubt it, since the method calls below assume it.)

Reply:
I bet I need to move and check line 44. I don't think the accessor is being set.